### PR TITLE
Hydronic: new fields, abstracts and equipment types

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by evgeniya-engel** [comment]
@shambergoldstein @cquigley2 ready for your review! Thank you!

File: ontology/yaml/resources/HVAC/entity_types/CH.yaml [hunk 1]
```diff
@@ -662,6 +690,25 @@ CH_SS_SCHWTC_CDWFRM_CHWSWISOVM_CWSISOVM_CHWDPM_CHWFRM_CWDT_REFPM_IGM_DXSWC:
   - IGM
   - DXSWC
 
+# 2081 chillers
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/CH.yaml:693]: remove comment

File: ontology/yaml/resources/HVAC/entity_types/CHWS.yaml [hunk 2]
```diff
@@ -133,6 +133,18 @@ CHWS_SWTC_WDPC_CLPM_WFRM:
   - CLPM
   - WFRM
 
+CHWS_SWTC_WDPC_CLPM_WFRM_MWFRM:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/CHWS.yaml:136]: How is makeup water incorporated in the CHWS? I would expect this to be on the CDWS

File: ontology/yaml/resources/HVAC/entity_types/CHWS.yaml [hunk 3]
```diff
@@ -525,6 +537,54 @@ CHWS_SS_WDPC_SWISOVPM_BYPVPM_WSEBYPVPM_PWDT_PRSWFRC_SESWTC_SEWFRM_CLPM:
...
+
+CHWS_SESWTC_SEWDPC_WSEBYPVM:
+  guid: "503d5173-65a9-422a-9788-c7c00f2196a2"
+  description: "Chilled water system with secondary supply water temperature and differential pressure control and waterside economizer bypass."
+  is_canonical: true
+  implements:
+  - CHWS
+  - SESWTC
+  - SEWDPC
+  - SEWFRM
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/CHWS.yaml:585]: Is `SEWFRM` supposed to be here? It's not on the canonical name

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**